sql/schema.sql

```diff
@@ -0,0 +1,16 @@
+-- create author table
+CREATE TABLE author (
+  id INT NOT NULL GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  first_name VARCHAR(255) NOT NULL,
+  last_name VARCHAR(255) NOT NULL  
+);
+
+-- create games table
+CREATE TABLE game (
+  id INT NOT NULL GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  publish_date DATE,
+  archived BOOLEAN,
+  multiplayer BOOLEAN,  
+  last_played_at DATE,  
+  FOREIGN KEY (id) REFERENCES item (id) ON DELETE CASCADE  
```

Comment:
When creating the tables I got this error `relation "item" does not exist` referencing this line `FOREIGN KEY (id) REFERENCES item (id) ON DELETE CASCADE  `

Reply:
create a table game updated,
thank you.
please see and give a feedback